### DISCUSSION TITLE
Disable the BlockedThreadChecker

### DISCUSSION THREAD

**Discussion by shachindrasingh** [apiman/apiman#2140] [Q&A] [answered]
how can we disable this option .. after one request this will block another request for some time ,

**Comment by msavy** [comment, marked as answer]
Hi 

Disabling the BlockedThreadChecker message will not actually prevent the blocking problem. 

You need to identify which custom functionality is causing the event loop's thread to be blocked.

**Reply by shachindrasingh** [reply]
we had configure the api. that api block for some time due to that . can we know how can we resolve this issue . i had use the default configuration.

**Reply by msavy** [reply]
You can change the BlockedThreadChecker with a System property, but the thread will still block and that entire event loop will wait until it is unblocked -- you'll just be hiding the problem in the logs.

`-Dvertx.options.maxEventLoopExecuteTime=<some big number of millis>`

Should only be used for debugging.
